From 07461ae0ce157922957407c59db77b50ba155efb Mon Sep 17 00:00:00 2001
From: Justin Maggard <jmaggard@netgear.com>
Date: Wed, 26 Oct 2016 20:18:41 +0000
Subject: [PATCH] connman.service: Disable DNS proxy

Disable DNS proxy by default.  It has issues with compression with some
DNS servers; it takes longer to time out DNS requests than using the
standard resolver; and on a couple occasions we've seen it hang, which
causes all sorts of problems on the system.
---
 src/connman.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/connman.service.in b/src/connman.service.in
index 7376346..a652b0a 100644
--- a/src/connman.service.in
+++ b/src/connman.service.in
@@ -11,7 +11,7 @@ Wants=network.target
 Type=dbus
 BusName=net.connman
 Restart=on-failure
-ExecStart=@sbindir@/connmand -n
+ExecStart=@sbindir@/connmand -n -r
 StandardOutput=null
 CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_BIND_SERVICE CAP_NET_RAW CAP_SYS_TIME CAP_SYS_MODULE CAP_SYS_ADMIN
 ProtectHome=true
-- 
2.1.4

